src/table/hooks/use-did-update-effect.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect, useRef } from 'react';
+
+export default function useDidUpdateEffect(effect: () => void, deps: React.DependencyList) {
+  const didMountRef = useRef(false);
+
+  useEffect(() => {
+    if (didMountRef.current) {
+      effect();
+    } else {
+      didMountRef.current = true;
+    }
+  }, deps);
```

Comment:
lint does not like the deps being added like this. I tried `[...deps]` and some other things but it says it is not an array literal no matter what. If there is a better way of doing this, pls let me know.

Reply:
yes, here the react-hooks lint cannot know what deps conclude, so it complains.
then I guess that we need to do something for the react-hooks lint

Reply:
I've disabled it, so it works at least

Reply:
It seems like you have to use the syntax that you have seperate variable that you put in the array, in the useEffect call. Thus we can't satisfy the rule since we don't know what the deps are. Spreading doesn't help either :/